source/common/json/config_schemas.cc

```diff
@@ -606,6 +606,7 @@ const std::string Json::Schema::ROUTE_ENTRY_CONFIGURATION_SCHEMA(R"EOF(
       "prefix_rewrite" : {"type" : "string"},
       "host_rewrite" : {"type" : "string"},
       "auto_host_rewrite" : {"type" : "boolean"},
+      "allow_websocket_upgrade" : {"type" : "boolean"},
```

Comment:
If this is set we actually **mandate** websocket upgrade, right? Otherwise we fail the request. The config field name should reflect that if possible, maybe just call it "websocket" or "use_websockets"?